Dockerfile

```diff
@@ -8,6 +8,7 @@ ARG CORS_ALLOWED_HEADERS=*
 ARG CORS_ALLOW_CREDENTIALS=false
 
 # Environment variables
+ENV TOMCAT_VERSION=$TOMCAT_VERSION
```

Comment:
one of the commands later in the Dockerfile uses TOMCAT_VERSION, but the ARG value doesn't propagate across multistage builds.

Reply:
Thx. We added this line here: https://github.com/geoserver/docker/pull/51